Makefile

```diff
@@ -0,0 +1,9 @@
+# -*- Makefile -*-
+
+all:
+	echo 'No rule for make all existing. Please run "sudo make install" to install seafctl'
+
+install: 
+	cp seafctl /usr/local/bin/seafctl
+	cp seafctl-completion.bash /usr/share/bash-completion/completions/seafctl
```

Comment:
Thanks for the patch!

Looks good overall, and makefile is a nice touch - I thought of packaging this next, but it didn't occur to me to create a makefile.

I'm thinking though, since it's not part of a package (hence installation to /usr/loca/ prefix) then perhaps a more compliant method would be to copy seafctl-completion.bash to /etc/bash_completion.d/ instead of messing with (generally) package manager controlled directories. Thoughts?

Reply:
You're right. Change it to `/etc/ bash_completion.d/seafctl`, that would be better.